app/webpack/src/modules/proceedings.js

```diff
@@ -1,126 +1,122 @@
-import $ from 'jquery'
-import axios from 'axios'
-import * as JsSearch from 'js-search'
-var typingTimer;                //timer identifier
-var doneTypingInterval = 1500;  //time in ms, 1.5 second for example
-var ariaText;
-
-async function getAll(){
-  const response = await axios.get('/v1/proceeding_types')
-  return response.data
+import axios from 'axios';
+import { hide, show, pluralize } from '../helpers';
+const doneTypingInterval = 500; // time in ms, decided on 400ms to make search seem instant (but it's not!)
```

Comment:
Really nitpicky, sorry, but the comment refers to 400ms when the actual interval is set as 500. Can we make them consistent?

Reply:
oops sorry I changed it but forgot to change the comment, will update now!